coralproject/talk
# [CORL-1282] Hide user media preferences if all media types are disabled

## What does this PR do?

If the admin's turn off all three of the media options (Giphy, Twitter, and YouTube) then we don't show the user the option to automatically unfurl all media.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Turn off all 3 media options under Admin > Config
- Go to your stream > My Profile > Preferences
- See that the unfurl media option is missing
- Turn one of the 3 options on under Admin > Config
- Refresh stream > My Profile > Preferences
- See that the unfurl media options is present
